src/Components/CustomAlert.js

```diff
@@ -1,25 +1,60 @@
 import React from "react"
-import { Stack, Alert } from "@mui/material"
+import { Alert } from "@mui/material"
+import { Slide } from "@mui/material"
 
 let StyledAlert = (props) => {
-	console.log("styledAlert props", props)
-	return (
-		<Stack spacing={2}>
-			<Alert
-				variant="filled"
-				severity="error"
-				sx={{ display: `${props.errorVisible}` }}
-			>
-				{props.message}
-			</Alert>
-		</Stack>
-	)
+	let variableForSlideInProp = props.errorVisible !== "none"
+	let errorDirection = "up"
+	if (props.errorVisible === "userCreated") {
+		//returns succuss alert of user being created
+		console.log("alert props", props)
+		return (
+			<>
+				<Slide
+					direction={`${errorDirection}`}
+					in={variableForSlideInProp}
+					mountOnEnter
+					unmountOnExit
+				>
+					<Alert
+						variant="filled"
+						severity="success"
+						sx={{ display: `${props.errorVisible}` }}
+					>
+						{`User successfully created! Please log in with email '${props.userDataObj.email}' and password '${props.userDataObj.password}'`}
+					</Alert>
```

Comment:
With this kind of thing, typically I would make styled alert a little more simple, and have it just display a message string as a prop, like you did below. Then, where this is used, change the message  `{`User successfully created! Please log in with email '${props.userDataObj.email}' and password '${props.userDataObj.password}'`}` 

That way, you can use this styled message for all kinds of things throughout the app. Instead of needing to add a bunch of cases for each message you want to display inside this component.

Reply:
Understanding the goal to have more interchangeable components throughout a project, but in this specific case I am not sure what you mean by 'this'? Do you mean switch from an if statement rendering the different messages depending on inputs, and instead have it always render the single component that is modified by the props passed to it?